CryptoCore/Services/Reddit/RedditClient.cs

```diff
@@ -0,0 +1,38 @@
+﻿using CryptoCore.Models.ApiModels;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace CryptoCore.Services.Reddit
+{
+    public class RedditClient
+    {
+        private readonly HttpClient _redditClient;
```

Comment:
Can we just name this _client instead of _redditClient? We know it's the reddit client due to it being here.
```suggestion
        private readonly HttpClient _client;
```